src/registrar/admin.py

```diff
@@ -2552,6 +2552,33 @@ def save_model(self, request, obj, form, change):
         super().save_model(request, obj, form, change)
 
 
+class PortfolioAdmin(ListHeaderAdmin):
+    # NOTE: these are just placeholders.  Not part of ACs (haven't been defined yet).  Update in future tickets.
+    list_display = ("organization_name", "federal_agency", "creator")
+    search_fields = ["organization_name"]
+    search_help_text = "Search by organization name."
+    # readonly_fields = [
+    #     "requestor",
+    # ]
+
+    def save_model(self, request, obj, form, change):
+        # ---- update creator ----
+        # Set the creator field to the current admin user
+        obj.creator = request.user if request.user.is_authenticated else None
```

Comment:
**blocking:** We do expect an organization to be changed multiple times, unlike with verified by staff. we need a check for if not obj.creator is null, otherwise it will overwrite existing values here